qdrant-landing/content/articles/what-is-quantization.md

```diff
@@ -0,0 +1,242 @@
+            "quantile": 0.99,  # Ignoring extreme outliers
+            "always_ram": True
+        }
+    }
+}
+```
+
+The **quantile** is used to calculate the quantization bounds. For example, if you specify 0.99 as the quantile, 1% of extreme values will be excluded from the quantization bounds. 
+
+This parameter only affects the resulting precision, not the memory footprint. You can tune it if you experience a significant decrease in search quality.
+
+The primary benefit of Scalar Quantization is **memory reduction.** It is especially useful for large-scale datasets, where memory and processing power become limiting factors. It also slighly improves performance, as distance calculations (such as dot product or cosine similarity) using int8 values are computationally simpler than using float32 values. 
+
+However, these performance gains are significantly lower compared to Binary Quantization, which we'll discuss later.
+
+# 2. What is Product Quantization?
+
+[Product Quantization](https://qdrant.tech/documentation/guides/quantization/#product-quantization) is a method used to compress high-dimensional vectors by representing them with a smaller set of representative points. 
+
+The process begins by setting up a **codebook,** which represents regions in the data space where common patterns occur. 
+
+The codebook in Qdrant is trained automatically during the indexation process. As vectors are added to the collection, Qdrant uses your specified quantization settings in the `quantization_config` to build the codebook and quantize the vectors. Here’s how you might set it up:
+
+```python 
+collection_config = {
+    "vectors": {
+        "size": 1024,
+        "distance": "Cosine"
+    },
+    "quantization_config": {
+        "product": {
+            "compression": "x32",
+            "always_ram": True
+        }
+    }
+}
+```
+
+Each region in the codebook is defined by a **centroid**, which serves as a representative point that summarizes the characteristics of that region. So, instead of treating every single data point as equally important, we can group similar points together and represent them with a single centroid that is more representative of the general characteristics of that group.
+
+The centroids used in Product Quantization are determined using the **[K-means clustering algorithm](https://en.wikipedia.org/wiki/K-means_clustering)**, which Qdrant applies to a representative sample of the dataset you want to quantize.
+
+
+![Codebook and Centroids example](/articles_data/what-is-vector-quantization/codebook.png)
+
+
+Qdrant always selects **K = 256** for the number of centroids in its implementation based on the fact that 256 is the maximum number of unique values that can be represented by a single byte.
+
+This makes the compression process efficient because each centroid index can be stored in a single byte.
+
+After the codebook is created, the original high-dimensional vectors are split into smaller **sub-vectors.** 
```

Comment:
The process description seems to be inaccurate. As far as I know, the codebook is created separately for each subvector, so we have different centroids for each segment. That has to be confirmed with the core team, but I think it makes sense to do it separately, as different segments of the original vectors may have different characteristics.

Reply:
You're right, that’s an important detail I hadn’t fully noticed before. I’ve applied the changes in the new commit